src/api.rs

```diff
@@ -48,6 +48,7 @@ pub struct Point {
   pub y: u16
 }
 
+/// Contain all the encoder configuration
```

Comment:
Can you add that this should contain settings which affect the encoded output? Any options which do not affect the output should be in `Config` instead.

Reply:
Sure :) 
Updated

Reply:
Done

Reply:
```


/// Contain all the encoder configuration which are taken for encoded output.
/// Encoder configurations which doesn't affect the encoded output are in 'config'.
#[derive(Clone, Debug)]
```